app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlController.kt

```diff
@@ -168,6 +168,18 @@ interface SessionControlController {
      * @see [CollectionInteractor.onCollectionMenuOpened] and [TopSiteInteractor.onTopSiteMenuOpened]
      */
     fun handleMenuOpened()
+
+    /**
+     * @see [ExperimentCardInteractor.onSetDefaultBrowserClicked]
+     */
+    fun handleSetDefaultBrowser() {
+    }
+
+    /**
+     * @see [ExperimentCardInteractor.onCloseExperimentCardClicked]
+     */
+    fun handleCloseExperimentCard() {
+    }
```

Comment:
nit: Is there a particular case why we want to have a default implementation?

Reply:
This is just to follow the pattern used in the app architecture.

Reply:
I asked we the other functions in this interface don't have default implementations.

Reply:
Oh, sorry, I get it now. I will remove it. :)